.pre-commit-config.yaml

```diff
@@ -42,7 +42,7 @@ repos:
   rev: v0.910
   hooks:
   - id: mypy
-    exclude: tests/data
+    exclude: ^tests/data|^tests/unit
```

Comment:
why ignore unit tests if they are also fully annotated?

Reply:
I can remove it - but when I run `pre-commit` on my tests, there were lots of mypy errors (unrelated to my changes). I am not sure how CI tests are not failing it - I guessed it's because tests are excluded (but I have not checked that). I am happy to remove it as it is anyhow not really related to the change and if needed - should be added in separate PR. It's just  a small annoyance for running changes locally on pre-commit  - I  have to add `--no-verify` but this is really a small annoyance.

Reply:
Resolved. I removed the change.

Reply:
Yeah. I know all about it (and more). I personally introduced pre-commit to apache airlfow ~ 2 years ago and we have 90+ pre-commits out of which I developed or integrated probably 30% of. Including pre-commits that check that pre-commit list in our documentatio is updated https://github.com/apache/airflow/blob/main/STATIC_CODE_CHECKS.rst#available-pre-commit-checks

Reply:
It is likely, I have different mypy version in this case or I have different packages installed or both.

BTW. unless you have very "tightly controlled" environment, there is no way to make sure that different users will have the same error list. Te list of errors returned MyPy results are very sensitive to:

* python version used
* list of packages installed 
* including which typeshed packages are installed https://github.com/python/typeshed

In case of airflow - we make sure of all those because we run mypy inside a docker image which has "currently blessed" set of dependencies including MyPy version (and we always use Python 3.7 as a base).

For PyPI that's likely not needed  as you have far less number of dependencies, but I think it might be because you only pin few dependencies in mypy pre-commit.  I might be able to take a look at it and maybe attempt to make sure that the env is exactly the same for everyone in `pip` (but it might actually need a custom hook).

Reply:
BTW. When I run it from within venv that has 'pip -e .' installed (including all test dependencies), I have no errors. So indeed it depends on list of packages installed.